docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+services:
+  db:
+    image: postgres
+    ports:
+      - "5432:5432"
+    expose:
+      - 5432
```

Comment:
Why is DB's port exposed? If DB is part of backend, frontend doesn't need to know the existence of DB, therefore frontend just needs to know the APIs of backend. Exposing backend port should be enough.

Reply:
This is meant for development use case: developer will be able to access the postgres database directly from host machine, for example, using PgAdmin to connect.
In production use case, though this hasn't been tested yet, we wouldn't need the postgres service because we'll use RDS instead. My opinion is to leave it there for the ease of development.